### ISSUE TITLE
[Bug]: $29m loss and high confidence?

### ISSUE THREAD

**Issue by codalectic** [TRADEBETTERAPP/tradebetter-public#121]
### Describe the bug

<img width="894" height="1062" alt="Image" src="https://github.com/user-attachments/assets/ce75b780-724c-423f-9d4c-ed9c9447f8c0" />

Don't think this is right

**Comment by Ankit-0202** [comment]
Fixed in `issues` branch (backend commit 9f45e86). Added hard penalty multiplier in confidence scoring for wallets with severely negative total PnL: >10k loss = 55% cap, >100k = 30% cap, >1M = 15% cap on track record score. A $29M loss wallet will now score very low confidence regardless of other factors.

**Comment by Ankit-0202** [comment]
Fixed and deployed. Wallets with big losses now get penalized in the confidence score, so you won't see high-confidence signals from wallets that are down millions. The worse the track record, the lower the confidence — a $29M loss wallet will now always show low confidence.
